Tests/Data/HydroMechanics/Linear/Confined_Compression/cube_1e3.prj

```diff
@@ -59,6 +61,18 @@
                     </properties>
                 </phase>
             </phases>
+            <properties>
+                <property>
+                    <name>reference_temperature</name>
+                    <type>Constant</type>
+                    <value>0</value>
```

Comment:
This is rather cold environment...
Should be mentioned at least in the PR. Better would be something around 20 degree Celsius.

Reply:
Okay, then I go with the 293.15 K for all tests